docs/reference/extensibility/nuget-exe-Credential-Providers.md

```diff
@@ -9,15 +9,11 @@ ms.topic: conceptual
 
 # Authenticating feeds with nuget.exe credential providers
 
-*NuGet 3.3+*
+In version `3.3` support was added for `nuget.exe` specific credential providers. Since then, in version `4.8` [support for credential providers](NuGet-Cross-Platform-Authentication-Plugin.md) that work across all command line scenarios (`nuget.exe`, `dotnet.exe`, `msbuild.exe`) was added.
 
-When `nuget.exe` needs credentials to authenticate with a feed, it looks for them in the following manner:
+See [Consuming Packages from authenticated feeds](../../consume-packages/consuming-packages-authenticated-feeds.md#nugetexe) for more details on all authentication approaches for `nuget.exe`
```

Comment:
looks like a broken link https://github.com/NuGet/docs.microsoft.com-nuget/pull/1931#issuecomment-592772300

Reply:
That's not the broken link, it's a different one, the comment explains it.